### PR TITLE
Add app configuration in .conf file for mlflow

### LINKED ISSUE #40: Define prinz configuration using env variables
Use application.conf file to define the environment of application which includes maily mlflwo configurtion

### PR DESCRIPTION
**Pull request by avan1235**
Closes #40

### CODE REVIEW

**Comment by avan1235** [comment]
Closes #40

File: dev-environment/mlflow/run.sh [hunk 1]
```diff
@@ -9,14 +9,11 @@ sleep 5
...
+echo "Mlflow run finished" && \
 RUN_ID="$(cat run.out | tail -n 1 | grep -oP "(?<=').*?(?=')")" && \
-rm -f run.out
-
-echo "Mlflow run finished"
-echo "Mlflow run id: $RUN_ID"
-
-echo "Serving trained model in Mlflow registry on port $MODEL_PORT"
-
+rm -f run.out && \
```
Comment (Jadw1) [inline dev-environment/mlflow/run.sh:15]: why this commands and connected (&&) together?

Comment (avan1235) [reply dev-environment/mlflow/run.sh:15]: @Jadw1  it's a concise way to say "if any of the previous commands failed the next one won't execute and non zero code is returned"

Comment (Jadw1) [reply dev-environment/mlflow/run.sh:15]: oh, fair point

**Review by Jadw1 — APPROVED** [review]

**Pull request merged**